### PR TITLE
feature/tasks

### PR DESCRIPTION
**Pull request by HolyHol**

### CODE REVIEW

File: app/controllers/api/v1/tasks_controller.rb [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-                .ransack(ransack_params)
-                .result
-                .page(page)
-                .per(per_page)
-  
+    tasks = Task.all.
+      ransack(ransack_params).
+      result.
+      page(page).
+      per(per_page)
```
Comment (nut4k1) [inline app/controllers/api/v1/tasks_controller.rb:7]: nonono, previous variant is better

File: app/javascript/components/ColumnHeader/ColumnHeader.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  const handleLoadMore = () => onLoadMore(id, currentPage + 1);
+
+  return (
+    <div className={styles.root}>
+      <div className={styles.title}>
+        <b>{title}</b> ({count}/{totalCount || '…'})
+      </div>
+      {count < totalCount && (
+        <div className={styles.actions}>
+          <IconButton aria-label="Load more" onClick={() => handleLoadMore()}>
```
Comment (nut4k1) [inline app/javascript/components/ColumnHeader/ColumnHeader.js:30]: onClick={handleLoadMore}

File: app/javascript/components/Task/Task.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+const Task = ({ task, onClick }) => {
+  const styles = useStyles();
+
+  const handleClick = () => onClick(task);
+    const action = (
+      <IconButton onClick={handleClick}>
+        <EditIcon />
+      </IconButton>
+    );
```
Comment (nut4k1) [inline app/javascript/components/Task/Task.js:21]: it seem's like extra padding

**Review by nut4k1 — APPROVED** [review]
good!

File: app/controllers/api/v1/tasks_controller.rb (hunk 1 above)
Comment (HolyHol) [reply app/controllers/api/v1/tasks_controller.rb:7]: ok, I fix all mistakes in the next pr.

**Pull request merged**